src/io/openshift/plugins/analytics.groovy

```diff
@@ -5,6 +5,37 @@ import io.openshift.Events
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime() ?: 'node'
+        spawn (image: image, version: config.version()) {
+          echo "Pod spawned."
+          def url = Utils.shWithOutput(this, "git config remote.origin.url")
+          echo "URL:$url"
+          image = config.runtime() ?: 'maven'
+          echo "Image:$image"
+          
+          if(image == "node") {
+              sh '''
+                  echo "running additional npm commands"
+                  npm install --prod
```

Comment:
did you try `npm install --production --package-lock-only ` ? doesn't that suffice?